### PR TITLE
Rename set/getCameraShakeLevel to set/getCameraDrunkLevel 

### PR DESCRIPTION
**Pull request by Unde-R**
In addition to what Lpsd said in #1197 I renamed:
Both ( setCameraShakeLevel & getCameraShakeLevel ) to ( setCameraDrunkLevel & getCameraDrunkLevel )

### CODE REVIEW

**Comment by qaisjp** [comment]
There are other things you need to do when renaming functions, like:

- keeping the original name around
- marking the old name as deprecated, so it can be auto upgraded

I'll write some documentation for this and send you a link here.

**Comment by Unde-R** [comment]
Sure thing mate

**Comment by qaisjp** [comment]
Instructions added here: https://github.com/multitheftauto/mtasa-blue/wiki/Lua-API#how-to-rename-functions

**Comment by Unde-R** [comment]
Feel free to request any other changes ^^

**Comment by Unde-R** [comment]
https://github.com/multitheftauto/mtasa-blue/blob/15b2ec72367c1a4226b5afebb7d38b113ca34022/Server/mods/deathmatch/logic/CResourceChecker.Data.h#L211
How I can get the current version so I can add it here?

**Comment by qaisjp** [comment]
This is the number that appears on https://buildinfo.mtasa.com.

The number increments per-commit, but only appears on the website after the next nightly is built.

So, unfortunately this is multi-step. (Which is why everyone forgets about it.)

This one needs to get merged, then you need to wait a day, and then you need to send a PR with the version number.

(But we probably won't merge this before 1.5.8? I don't know.)

**Comment by Unde-R** [comment]
Ah alright ty

**Review by ghost — DISMISSED** [review]

File: Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h [hunk 1]
```diff
@@ -24,7 +24,7 @@ class CLuaCameraDefs : public CLuaDefs
     LUA_DECLARE(GetCameraTarget);
     LUA_DECLARE(GetCameraInterior);
     LUA_DECLARE(GetCameraGoggleEffect);
-    LUA_DECLARE(GetCameraShakeLevel);
+    LUA_DECLARE(GetCameraDrunkLevel);
```
Comment (ghost) [inline Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h]: Consider using the new argument parser for this.

File: Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h [hunk 2]
```diff
@@ -37,7 +37,7 @@ class CLuaCameraDefs : public CLuaDefs
     LUA_DECLARE(GetCameraClip);
     LUA_DECLARE(SetCameraViewMode);
     LUA_DECLARE(SetCameraGoggleEffect);
-    LUA_DECLARE(SetCameraShakeLevel);
+    LUA_DECLARE(SetCameraDrunkLevel);
```
Comment (ghost) [inline Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h]: +

File: Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h (hunk 1 above)
Comment (Unde-R) [reply Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h]: Sure mate, tmrw

**Review by StrixG — DISMISSED** [review]

**Comment by Pirulax** [comment]
I'm not quite sure whats the problem with `shake level`?

**Comment by Unde-R** [comment]
> I'm not quite sure whats the problem with `shake level`?

Well gta named it drunkLevel

**Comment by Unde-R** [comment]
thnx for the commits @StrixG

**Comment by Pirulax** [comment]
Well okay. I'm not quite sure if the old function needs to be deprecated tho.
They both achieve the same thing. Its like `std::string::length()` and `std::string::size()` they achieve the same thing, but have different names for convenience.
Perhaps we could keep both functions?

**Review by patrikjuvonen — APPROVED** [review]

**Pull request merged**